WcaOnRails/app/controllers/translations_controller.rb

```diff
@@ -49,36 +49,38 @@ def update
     @pr_url = Octokit.create_pull_request(upstream_repo, "master", "#{user_login}:#{branch_name}", message, pr_description_for(current_user, locale))[:html_url]
   end
 
+  # rubocop:disable Style/NumericLiterals
```

Comment:
Out of curiosity, what does rubocop *want* you to do here instead?

Reply:
Same as above :smiley: